pkg/scrape/realitylovers.go

```diff
@@ -13,18 +13,16 @@ import (
 	"gopkg.in/resty.v1"
 )
 
-func RealityLovers(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out chan<- models.ScrapedScene) error {
+func RealityLoversSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out chan<- models.ScrapedScene, scraperID string, siteID string, URL string) error {
 	defer wg.Done()
-	scraperID := "realitylovers"
-	siteID := "RealityLovers"
 	logScrapeStart(scraperID, siteID)
 
-	sceneCollector := createCollector("realitylovers.com")
+	sceneCollector := createCollector("realitylovers.com", "tsvirtuallovers.com")
 
 	sceneCollector.OnHTML(`html`, func(e *colly.HTMLElement) {
 		sc := models.ScrapedScene{}
 		sc.SceneType = "VR"
-		sc.Studio = "RealityLovers"
+		sc.Studio = siteID
```

Comment:
Don't change this line. The studio is the same for both.

```suggestion
		sc.Studio = "RealityLovers"
```

Reply:
Oops, thank you. Fixed!